### PR TITLE
remove browser sync

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/server.js [hunk 1]
```diff
@@ -16,19 +15,9 @@ Object.keys(config).map((configKey) => {
...
-    browserSync({
-      files: ['./views/**/*.*', '../public/**/*.*'],
-      notify: true,
-      open: false,
-      port: config.port,
-      proxy: `localhost:${config.port}`,
-      ui: false,
-    });
-  });
+  logger.info(` Marketing Pages - \x1b[35m${config.appBaseUri}\x1b[0m`);
```
Comment (alibombaye) [inline app/server.js]: What the ???

Comment (kat1906) [reply app/server.js]: makes it pink 😄

Comment (kat1906) [reply app/server.js]: and a clickable link for the lazy ones

Comment (kat1906) [reply app/server.js]: ie me

Comment (alibombaye) [reply app/server.js]: ooh fancy. OK mmm in that case can we for marketing pages make it navigate to `{config.appBaseUri}/supplier/solution/100000-001` then it'll be a clickable link that works 😂

Comment (kat1906) [reply app/server.js]: YES! 🎉

**Pull request merged**